frontend/src/apiAccess/apiApps.js

```diff
@@ -57,6 +57,8 @@ const FETCH_USER_ACTIONS_CARDS_COMPLETED_PAGE =
 
 const FETCH_ACTIVITY_LOG_DAY_COUNTS_PAGE = "FETCH_ACTIVITY_LOG_DAY_COUNTS_PAGE";
 
+const ACTIVITY_LOG_ENTRIES = "ACTIVITY_LOG_ENTRIES";
```

Comment:
Is this meant to be a verb or a noun? Surely we are trying to fetch the activity log entries? This is the Action name, right?